test/unittests/t_fs_traversal.cc

```diff
@@ -26,7 +27,10 @@ class T_FsTraversal : public ::testing::Test {
       File,
       Symlink,
       Untouched,
-      Unspecified
+      Unspecified,
+      Socket,
+      BlockedDevice,
```

Comment:
Not a show stopper for the merging, but it is actually a "block device" (because data is transferred in blocks, it's not blocked).